demos/basics/loading-networkx.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<blockquote><a href="https://networkx.github.io/" target="_blank">NetworkX</a>&nbsp;is a library for working with graphs that provides many convenient I/O functions, graph algorithms and other tools for working with graphs.</blockquote><p>"for working with graphs" is repeated, maybe remove one of them?</p><p> </p><p>For the bullet points listing the different sections:</p><ul><li>some things are italicised and others not - not sure if this was on purpose but looks a bit inconsistent to me.</li><li>maybe the surrounding double quotes for each of the sections are not so necessary?</li><li>thoughts on the word "direction" instead of "orientation" of the edge? Orientation is used later in the notebook to describe edge types instead</li></ul><p> </p><blockquote>StellarGraph supports loading data from many sources with all sorts of data pre-processing, because it can load from&nbsp;<a href="https://pandas.pydata.org/" target="_blank">Pandas</a>&nbsp;DataFrames and from . This notebook demonstrates loading data from Pandas. Loading from Pandas is more efficient and more convenient for pre-processing (some of which is demonstrated), and is thus encouraged. See&nbsp;<a href="https://github.com/stellargraph/stellargraph/raw/413bf1e978897c864ce9143e6859cefb440a1358/demos/basics/loading-networkx.ipynb" target="_blank">the "Loading data into StellarGraph from NetworkX" demo</a>&nbsp;for more details about the specifics of loading data from a NetworkX object.</blockquote><p>I think this belongs in the pandas notebook instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
> "for working with graphs" is repeated, maybe remove one of them?

👍 

> some things are italicised and others not - not sure if this was on purpose but looks a bit inconsistent to me.

The intention is that new/not-yet-defined words are italicised, but it does end up looking weird. I'll remove.

> maybe the surrounding double quotes for each of the sections are not so necessary?

The intention here was that that's the thing to search for to jump to the relevant section. Maybe over the top, though.

> thoughts on the word "direction" instead of "orientation" of the edge? Orientation is used later in the notebook to describe edge types instead

I decided against direction, because it seemed like defining the word it terms of itself "directed ... direction". Maybe I'll switch to something like "which end of the edge is the 'start' and which is the 'end' is significant".

> I think this belongs in the pandas notebook instead?

Haha, oops.